cmd/soroban-rpc/internal/methods/get_version_info.go

```diff
@@ -0,0 +1,68 @@
+package methods
+
+import (
+	"context"
+	"github.com/creachadair/jrpc2"
+	"github.com/creachadair/jrpc2/handler"
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/soroban-rpc/cmd/soroban-rpc/internal/config"
+	"github.com/stellar/soroban-rpc/cmd/soroban-rpc/internal/daemon/interfaces"
+	"github.com/stellar/soroban-rpc/cmd/soroban-rpc/internal/db"
+)
+
+type GetVersionInfoRequest struct {
```

Comment:
if a json rpc endpoint doesn't take any arguments you can just omit the request entirely. see cmd/soroban-rpc/internal/methods/get_latest_ledger.go for example